src/main/javascript/acd/ol/ol.custommap.js

```diff
@@ -116,7 +116,6 @@ acd.ol.CustomMap.prototype.addBaseLayerAndOverlayMapLayer = function(baseLayer,
 												overviewMapLayers: [overlayMapLayer]
 											}
 										});
-		this.addControl(this.overviewMapControl); 
```

Comment:
De overview map wordt niet meer toegevoegd, maar wat met de gebruikers die wel een overview map willen? De mogelijkheid toevoegen om de overview map te enable en eventueel weer te disablen lijkt me nodig te zijn.

Reply:
De toevoegen zal toch gebeuren via vl-ui-overviewmap? 
https://github.com/milieuinfo/webcomponent-vl-ui-map/pull/75

Reply:
Inderdaad, maar ik dacht dat daarvoor een API uitbreiding moest gebeuren zodat de vl-ui-overviewmap die kon aanroepen. Ik heb de PR op vl-ui-overviewmap nog niet bekeken, dan is het vermoedelijk in orde en mag je dit negeren.